rest/main.go

```diff
@@ -13,6 +13,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/couchbase/gocb/v2"
```

Comment:
Generally we're trying to avoid having gocb-specific information in anywhere but the base package, to facilitate future changes to SDK versions or Bucket implementations. In this case I think it's preferable to pass the config.BucketCredentials down to NewCouchbaseCluster.